engine/vcs/gitlab/client_pull_request.go

```diff
@@ -6,3 +6,7 @@ import "github.com/ovh/cds/sdk"
 func (c *gitlabClient) PullRequests(string) ([]sdk.VCSPullRequest, error) {
 	return []sdk.VCSPullRequest{}, nil
 }
+
+func (g *gitlabClient) PullRequestComment(string, int, string) error {
```

Comment:
receiver name g should be consistent with previous receiver name c for gitlabClient